internal/fsext/fileutil.go

```diff
@@ -89,8 +41,6 @@ func SkipHidden(path string) bool {
 		"obj":              true,
 		"out":              true,
 		"coverage":         true,
-		"tmp":              true,
-		"temp":             true,
```

Comment:
tempdirs used on tests on linux usually are in `/tmp`, so the grep/glob tests were not working properly.

not sure if we really want this here or not, if we do, will have to find another way...

long term I think should remove this function entirely as those files should be ignored specifically by gitignore/crushignore.

Having some files magically ignored might be frustrating to users that don't expect it...